wallets/lnbits/client.js

```diff
@@ -8,7 +8,7 @@ export async function testSendPayment ({ url, adminKey, invoiceKey }, { logger }
   url = url.replace(/\/+$/, '')
   await getWallet({ url, adminKey, invoiceKey })
 
-  logger.ok('wallet found')
+  logger.info('wallet found')
```

Comment:
I didn't want this to affect the wallet status

Reply:
I reverted this and instead use `context.status` now to determine which log messages should determine wallet status. This is only set for payments (= `context.bolt11` is set).